PWGCF/TwoParticleCorrelations/Tasks/lambdaR2Correlation.cxx

```diff
@@ -239,21 +262,21 @@ struct lambdaCorrTableProducer {
   }
 
   template <typename T>
-  bool selProton(T const& track)
+  bool selPIDTrack(T const& track, float tpcNsigma, float tofNsigma)
```

Comment:
For the next iteration I would suggest to have an `enum` with the type of track you want to identify and pass it as one function parameter. Then using a `switch` on the type of track store in local variables the corresponding `tpcNsigma` and `tofNsigma` values before proceeding with the rest of the function

It is just a suggestion but I believe it will make the function call clearer an less prone to mistakes